test/extended/networking/network_segmentation.go

```diff
@@ -945,13 +959,14 @@ var _ = Describe("[sig-network][OCPFeatureGate:NetworkSegmentation][Feature:User
 	})
 })
 
+var nadToUdnParams = map[string]string{
+	"primary":   "Primary",
+	"secondary": "Secondary",
+	"layer2":    "Layer2",
+	"layer3":    "Layer3",
+}
```

Comment:
Actually to the format we use in UDN/CUDN API (pascal case).
They come to have the same names, which are single word so it seems capitulated :nerd_face:

Reply:
Asking in case it made sense to just pass a func to map to all values that would capitalize the first character. 

Not sure it makes sense - keep what you have.